src/mcts/search.h

```diff
@@ -100,12 +100,13 @@ class Search {
   // Computes the best move, maybe with temperature (according to the settings).
   void EnsureBestMoveKnown();
 
+  class TerminalScore;
   // Returns a child with most visits, with or without temperature.
   // NoTemperature is safe to use on non-extended nodes, while WithTemperature
   // accepts only nodes with at least 1 visited child.
   EdgeAndNode GetBestChildNoTemperature(Node* parent) const;
-  std::vector<EdgeAndNode> GetBestChildrenNoTemperature(Node* parent,
-                                                        int count) const;
+  std::vector<TerminalScore> GetBestChildrenNoTemperature(Node* parent,
```

Comment:
Does this mean that `GetBestChildNoTemperature` can only return terminal nodes? That doesn't sound right to me.

Reply:
Reverted to just EdgeAndScore.